zerver/webhooks/buildbot/doc.md

```diff
@@ -0,0 +1,18 @@
+Zulip supports integration with Buildbot and can notify you of
+your build statuses.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Edit the buildbot configuration file to add a new Zulip reporter
+ ([or follow steps listed here](http://docs.buildbot.net/latest/manual/configuration/reporters.html#zulipstatuspush)):
+
+        from buildbot.plugins import reporters
+
+        zs = reporters.ZulipStatusPush(endpoint='your-organization@zulipchat.com',
+                                       token='API_key', stream='stream_to_post_in')
+        c['services'].append(zs)
+
+{!congrats.md!}
```

Comment:
Hmm.. I'm confused, why did you ask the user to construct a webhook URL? It seems like it isn't really necessary here and that the user can just create a bot and supply the API key and stream name when setting up the reporter, right?

Reply:
Are you talking about in step 2? I did that because the user needs to create a bot to get the API key but that document also tells the user to construct a URL. I didn't want to mess up the consistency and used the existing create-bot-construct-url-indented.md. Sorry about that.

Reply:
Yes, step 2, we don't need a webhook URL here. I think it would be better to just use the `create-a-bot.md` macro here (I could be wrong about the name, they all live in `templates/zerver/help/include/`) and then simply asking the user to provide the bot's API key in step 3. :)

Reply:
Sorry about that. I knew that it didnt require the whole URL but I thought that I should be maintaining consistency across the docs and I didn't properly look for a template which documented only creating the bot. It should be good to go now.